java/test/src/main/java/org/ray/api/test/CrossLanguageInvocationTest.java

```diff
@@ -62,12 +67,22 @@ public void testCallingPythonActor() {
     RayPyActor actor = Ray.createPyActor(PYTHON_MODULE, "Counter", "1".getBytes());
     RayObject res = Ray.callPy(actor, "increase", "1".getBytes());
     Assert.assertEquals(res.get(), "2".getBytes());
+    // Call a python function which create a python actor
+    // and pass the actor handle to callPythonActorHandle.
+    res = Ray.callPy(PYTHON_MODULE, "py_func_pass_python_actor_handle");
+    Assert.assertEquals(res.get(), "3".getBytes());
```

Comment:
Make this a separate test case? e.g., `testPassActorHandleFromPythonToJava`

Reply:
OK. I separated test cases to `testPassActorHandleFromPythonToJava` and `testPassActorHandleFromJavaToPython`.